lib/cli/index.js

```diff
@@ -77,6 +65,10 @@ function configureLogger(env) {
 
 // Options: Array cliArgs, Stream inputStream, Stream outputStream
 module.exports = function(options) {
+  // work around misbehaving libraries,
+  // so we can correctly cleanup before actually exiting.
+  willInterruptProcess.capture(options.process || process);
```

Comment:
ember test helper requires that the `process` has an ability to be captured on each cli invocation.
If we leave capturing of the `process` outside of invocation in the top of the file like it was before we will loose an ability to configure(capture/free) the process between test runs.